nmrfx-core/src/main/java/org/nmrfx/chemistry/relax/RelaxationData.java

```diff
@@ -25,21 +25,31 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.util.*;
+import java.util.stream.Collectors;
 
 /**
  *
  * @author mbeckwith
  */
 public class RelaxationData implements RelaxationValues {
 
+    public String getId() {
+        return id;
+    }
+
+
+    public ResonanceSource getResSource() {
+        return resSource;
+    }
+
```

Comment:
This one seems like it might be redundant? What is the reason to have both `getResSource` and `getResonanceSource`?